net.pdl2ork.PdL2Ork.yml

```diff
@@ -0,0 +1,199 @@
+id: net.pdl2ork.PdL2Ork
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
```

Comment:
```suggestion
runtime-version: '23.08'
```

Reply:
Please see my comment. I would prefer if we left it at 22.08, as testing 23.08 will require a good month, given the complexity of the build process and its reliance on a number of other submodules.